tools/independent/RuntimeGraphGenerator/Program.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
the reason they had to move was to add the global.json in the `independent` directory. I thought the name was fitting as these are the tools which leverage the global.json to be independent of project references. Admittedly it's not the best name though, so if you have a suggestion feel free.